app/api/export-csv/input-validation.ts

```diff
@@ -31,7 +31,14 @@ export const exportCsvSchema = z.object({
     .max(100)
     .optional(),
   legalCategories: z
-    .array(z.string().regex(/^\d{4}$/, 'Must be a 4-digit code'))
+    .array(
+      z
+        .string()
+        .regex(
+          /^(\d{4}|\d\*)$/,
+          'Must be a 4-digit code or a digit followed by "*" (e.g., "1*")'
```

Comment:
Is this the error that the user will see?

Reply:
We only display Validation error if this happens (but the frontend doesn't allow it).